apps/ui-playground/.source/index.ts

```diff
@@ -0,0 +1,217 @@
+    },
+    data: components_11,
+  },
+  {
+    info: {
+      path: "components/dialog.mdx",
+      absolutePath: "/Users/sean/code/cal.com/apps/ui-playground/content/design/components/dialog.mdx",
+    },
+    data: components_12,
+  },
+  {
+    info: {
+      path: "components/dropdown.mdx",
+      absolutePath: "/Users/sean/code/cal.com/apps/ui-playground/content/design/components/dropdown.mdx",
+    },
+    data: components_13,
+  },
+  {
+    info: {
+      path: "components/emptyScreen.mdx",
+      absolutePath: "/Users/sean/code/cal.com/apps/ui-playground/content/design/components/emptyScreen.mdx",
+    },
+    data: components_14,
+  },
+  {
+    info: {
+      path: "components/form.mdx",
+      absolutePath: "/Users/sean/code/cal.com/apps/ui-playground/content/design/components/form.mdx",
+    },
+    data: components_15,
+  },
+  {
+    info: {
+      path: "components/input.mdx",
+      absolutePath: "/Users/sean/code/cal.com/apps/ui-playground/content/design/components/input.mdx",
+    },
+    data: components_16,
+  },
+  {
+    info: {
+      path: "components/navigation.mdx",
+      absolutePath: "/Users/sean/code/cal.com/apps/ui-playground/content/design/components/navigation.mdx",
+    },
+    data: components_17,
+  },
+  {
+    info: {
+      path: "components/pagination.mdx",
+      absolutePath: "/Users/sean/code/cal.com/apps/ui-playground/content/design/components/pagination.mdx",
+    },
+    data: components_18,
+  },
+  {
+    info: {
+      path: "components/rangeslider.mdx",
+      absolutePath: "/Users/sean/code/cal.com/apps/ui-playground/content/design/components/rangeslider.mdx",
+    },
+    data: components_19,
+  },
+  {
+    info: {
+      path: "components/select.mdx",
+      absolutePath: "/Users/sean/code/cal.com/apps/ui-playground/content/design/components/select.mdx",
+    },
+    data: components_20,
+  },
+  {
+    info: {
+      path: "components/switch.mdx",
+      absolutePath: "/Users/sean/code/cal.com/apps/ui-playground/content/design/components/switch.mdx",
+    },
+    data: components_21,
+  },
+  {
+    info: {
+      path: "components/tabs.mdx",
+      absolutePath: "/Users/sean/code/cal.com/apps/ui-playground/content/design/components/tabs.mdx",
+    },
+    data: components_22,
+  },
+  {
+    info: {
+      path: "components/textarea.mdx",
+      absolutePath: "/Users/sean/code/cal.com/apps/ui-playground/content/design/components/textarea.mdx",
+    },
+    data: components_23,
+  },
+  {
+    info: {
+      path: "components/toast.mdx",
+      absolutePath: "/Users/sean/code/cal.com/apps/ui-playground/content/design/components/toast.mdx",
+    },
+    data: components_24,
+  },
+  {
+    info: {
+      path: "components/toggleGroup.mdx",
+      absolutePath: "/Users/sean/code/cal.com/apps/ui-playground/content/design/components/toggleGroup.mdx",
+    },
+    data: components_25,
+  },
```

Comment:
is this file intended to be added to the repo? or a missing item in .gitignore?

Reply:
Hmmm. I think its missing from git ignore will fix